src/LazyRender.jsx

```diff
@@ -111,6 +114,19 @@ var LazyRender = React.createClass({
     });
   },
 
+  componentDidUpdate: function() {
+    //important to update the child height in the case that the children change(example: ajax call for data)
+    if(this.state.childHeight !== this.getChildHeight())
+      this.setState({childHeight: this.getChildHeight()});
```

Comment:
Not sure what you mean by cascading renders. It would cause another render, but stop at the next componentDidUpdate because the childHeight didn't change.